### PR TITLE
Add missing xml doc to framework classes

### PR DESCRIPTION
**Pull request by Jjagg**
This was a relaxing experience up to some point, but then it got tedious and I decided to stop.

### CODE REVIEW

**Comment by Jjagg** [comment]
Woops, looks like a bug in SharpDoc!

File: MonoGame.Framework/Game.cs [hunk 1]
```diff
@@ -241,16 +264,28 @@ public TimeSpan TargetElapsedTime
             }
         }
 
+        /// <summary>
+        /// Indicates if this game is running with a fixed time between frames.
+        /// When set to <code>true</code> the time target time between frames is
```
Comment (Jjagg) [inline MonoGame.Framework/Game.cs]: Typo here

File: MonoGame.Framework/Game.cs [hunk 2]
```diff
@@ -241,16 +264,28 @@ public TimeSpan TargetElapsedTime
...
+        /// Get a container holding service providers attached to this <see cref="Game"/>.
+        /// </summary>
         public GameServiceContainer Services {
             get { return _services; }
         }
 
+        /// <summary>
+        /// The <see cref="ContentManager"/> of this <see cref="Game"/>.
+        /// </summary>
+        /// <exception cref="ArgumentNullException">The </exception>
```
Comment (Jjagg) [inline MonoGame.Framework/Game.cs]: Woops

**Comment by Jjagg** [comment]
@tomspilman Any idea how to debug this? I ran into other issues trying to build the docs locally.

**Comment by tomspilman** [comment]
> Any idea how to debug this?

Actually running SharpDoc in the debugger should show what the error actually is.

I can do this, but I won't have time to do that for another week or so.

**Comment by Jjagg** [comment]
Thanks! No rush :)

**Comment by Jjagg** [comment]
@tomspilman Any chance you could look at this?

**Comment by tomspilman** [comment]
@Jjagg - Been too busy to look at this.  If you can fire up SharpDoc in the debugger it should be fairly easy to work out the bug and get a fix.

On the other hand we should contemplate looking at alternatives as in https://github.com/MonoGame/MonoGame/issues/5559.

**Comment by Jjagg** [comment]
I didn't manage to get SharpDoc working locally when I last tried debugging this issue, but if I have some time I'll take a look at docfx :)

**Comment by Jjagg** [comment]
@tomspilman @harry-cpp This should be good to go! Happy to finally get this build green 😎

**Pull request merged**